core/src/main/scala/org/broadinstitute/dsde/rawls/model/RawlsRequestContext.scala

```diff
@@ -0,0 +1,6 @@
+package org.broadinstitute.dsde.rawls.model
+
+import io.opencensus.trace.Span
+
+final case class RawlsRequestContext(userInfo: UserInfo, tracingSpan: Option[Span] = None)
+final case class RawlsTracingContext(tracingSpan: Option[Span] = None)
```

Comment:
There is one case where we have some tracing that does not have a user in context.